### PR TITLE
Russian translation of "archive"/"storyboard"/"transcriptions"

### PR DESCRIPTION
**Pull request by v1docq47**
hi! @lh1008 @erciccione
big update (and fix) for russian traslate ;)

translated:
- monero-outreach-docs\translations\ru\archive\dandelion_ru.md
- monero-outreach-docs\translations\ru\archive\getting_started_ru.md
- monero-outreach-docs\translations\ru\archive\loca_mesh_interview_ru.md
- monero-outreach-docs\translations\ru\archive\rpc_pay_ru.md
- monero-outreach-docs\translations\ru\storyboard\swot_ru.md
- monero-outreach-docs\translations\ru\storyboard\tool_for_renewal_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Lelantus_New_Protocol_for_Private_Transactions_with_Hidden_Origins_and_Amounts_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\The_Elephant_In_The_Pool_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Transaction_Efficiency_Then_And_Now_ru.md

fix:
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\ASIC-Resistant_Proof_of_Work_Fact_or_Fantasy_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Bringing_Privacy_Mainstream_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Challenges_in_Building_Software_with_Monero_Code_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Dandelion_Onions_Protecting_Transaction_Privacy_in_Monero_ru
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Monerokon_Madness_Schnorr_Schadness_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Network_Attacks_Obfuscation_Of_Payment_System_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Monerokon_Madness_Schnorr_Schadness_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Spam_Mitigation_The_Challenge_of_Private_and_Permissionless_ru.md
- monero-outreach-docs\translations\ru\transcriptions\monero-konferenco-2019\Visualizing_Monero_A_Figure_is_Worth_a_Thousand_Logs_ru.md

ping to review:
@LvMsterfild @TheFuzzStone

### CODE REVIEW

**Comment by lh1008** [comment]
Thank you @v1docq47, I'm currently reviewing to approve the PR.

File: monero-outreach-docs/translations/ru/storyboard/tool_for_renewal_ru.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+В 2009 году одним человеком или группой людей, выступившей под псевдонимом Сатоши Накамото, прямо по следам финансового кризиса был выпущен в свет предшественник Monero, названный Bitcoin. В генезис-блоке, первом блоке в блокчейне Bitcoin, Сатоши записал новостной заголовок, чтобы доказать время создания Bitcoin. Заголовок гласил: «The Times. 03 января 2009 г. Канцлер казначейства готов к повторному оказанию финансовой помощи банкам». Это оригинальное сообщение резонирует с сегодняшним днём, когда центральные банки стремятся поддержать мировую экономику, находящуюся в осаде.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/storyboard/tool_for_renewal_ru.md:13]: Hello @v1docq47,

I found a mistake in the English version, the first line of this paragraph, that can affect its meaning. Could you please check it out? The English version reads like this:

Wrong

- "In 2009 a pseudonymous person or group named Satoshi Nakamoto launched Monero’s precursor Bitcoin on the _**heals**_ of the great financial collapse."

Correct:

- "In 2009 a pseudonymous person or group named Satoshi Nakamoto launched Monero’s precursor Bitcoin on the _**heels**_ of the great financial collapse."

It was already fixed in PR #213.  

Thanks >:)

Comment (v1docq47) [reply monero-outreach-docs/translations/ru/storyboard/tool_for_renewal_ru.md:13]: @lh1008, tnx! everything is good :)

**Comment by v1docq47** [comment]
> 
> 
> @v1docq47 please next time use more meaningful commit messages. "translate+fix" is too generic :)

ok, no problem :)

**Comment by lh1008** [comment]
@v1docq47 I know this has already been closed but thunderosa said he will change his beer tonight for some vodka and that he feels a lot of grattitude of all the work you all do. 

Thank you very much

**Pull request merged**